aioesphomeapi/api.proto

```diff
@@ -996,6 +996,7 @@ message ListEntitiesClimateResponse {
   float visual_min_humidity = 24;
   float visual_max_humidity = 25;
   uint32 device_id = 26 [(field_ifdef) = "USE_DEVICES"];
+  bool requires_two_point_target_temperature = 27;
```

Comment:
https://github.com/esphome/esphome/pull/10987#discussion_r2400165202

Reply:
I like that approach a lot better. I seen in #264 Jesse straight-up changed a `bool` to a `uint32` -- would that be safe to do here, as well? Or better to just add a "flags" field and a comment to indicate that `supports_two_point_target_temperature` has been deprecated?

Reply:
Haha, sorry -- yes, the one here is new...but there is also the previously-existing flag on line 975, which this new flag complements. I was curious if they can be safely consolidated into a uint32 on line 975 without adding a new field here. 🙂

Reply:
I don't think it can be safety done as there may be other consumers of it.  We didn't have that concern with Bluetooth Proxy as there is no practical use outside of HA. I'd add a new flag and we can deprecate the old one at some point.